po/fr.po

```diff
@@ -5722,43 +5732,41 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:4
 msgid "Display a brief help message. Identical to the help action"
-msgstr ""
+msgstr "Afficher un bref message d’aide, Identique à l’action help"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:5
-#, fuzzy
 msgid "Remove any cached packages which can no longer be downloaded"
-msgstr "Nettoyer les paquets ne pouvant plus être téléchargés"
+msgstr "Nettoyer du cache les paquets ne pouvant plus être téléchargés"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:6
 msgid "Remove all downloaded .deb files from the package cache directory"
 msgstr "Supprimer tous les fichiers .deb du cache"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:7
-#, fuzzy
 msgid "Forget all internal information about what packages are 'new'"
-msgstr "Afficher l'état des fichiers extraits"
+msgstr ""
+"Oublier les informations à propos des paquets considérés comme 'nouveaux'"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:8
 msgid "Cancel all scheduled actions on all packages"
-msgstr ""
+msgstr "Annuler toutes les opérations prévues sur des paquets"
```

Comment:
"sur **les** paquets"? Could even be "sur *tous* les paquets."

Reply:
Basically, you're right, but using *des* instead of *les* highlights the fact that the actions to be cancelled were meant to be applied on packages only.